### PR TITLE
Adding more contributors to Contributor Wall of Fame

### PR DESCRIPTION
**Pull request by jasonblais**

### CODE REVIEW

File: site/content/contribute/team_contributions/_index.md [hunk 1]
```diff
@@ -9,22 +9,44 @@ Mattermost is an open source Slack-alternative that helps teams stay in sync, wi
...
...6.06.01), tgkouras (2019.09.11), theblueskies (2018.07.18), thekiiingbob (2019.03.07), therealpuneeth20 (2019.04.25), ThiefMaster (2018.06.19), thiyagaraj (2016.06.15), thomas9987 (2015.10.01), thomchop (2016.10.26), threepwood-mm (2019.08.29), timlyo (2016.02.19), tjuerge (2017.05.31), tmuwandi (2015.10.01), tomasmik (2019.06.08), tomitm (2015.10.06), tomocy (2019.01.12), toyorg (2015.10.28), trozz (2015.09.12), Tsynapse (2015.11.06), tylarb (2019.04.22), tyvsmith (2018.10.25), usmanarif (2016.05.03), uturkdogan (2018.05.15), vaithak (2018.11.05), valentijnnieman (2019.09.16), VeraLyu (2017.03.15), vinnymac (2015.11.01), viot (2015.10.02), vladikoff (2015.11.20), VolatianaYuliana (2019.05.07), vordimous (2018.02.07), Vorlif (2018.03.13), VPashkov (2018.10.24), waseem18 (2018.10.03), willdot (2019.06.06), willstacey (2015.08.27), Willyfrog (2019.06.15), wget (2016.10.17), Whiteaj36 (2017.08.03), Wipeout55 (2019.05.20), XinyueWang94 (2018.03.06), yeoji (2017.10.16), yth0625 (2017.11.07), yumenohosi (2016.05.04), yuvipanda (2015.11.01), Zackify (2015.08.29), Zaicon (2017.02.20), Zapix (2017.10.05), zeroimpl (2018.11.08), zetaab (2018.09.10), ZJvandeWeg (2017.01.09), zkry (2019.06.12)
```
Comment (justinegeffen) [inline site/content/contribute/team_contributions/_index.md:28]:
```suggestion
42wim (2015.10.23), a-arias (2019.07.06), aautio (2017.02.28), abdusabri (2019.09.24), abustany (2017.06.26), adamenger (2015.06.24), adarj (2019.09.19), adzimzf (2019.01.16), AGMETEOR (2019.07.08), Akash4927 (2018.10.03), akhilanandbv003 (2018.12.12), akihikodaki (2017.01.31), akrfjmt (2019.04.17), alanmoo (2016.02.29), alejandrosame (2019.07.07), alexander-akhmetov (2018.10.15), alexgaribay (2015.11.17), Alexgoodman7 (2018.02.08), alexrford (2017.07.02), ali-farooq0 (2019.03.28), alsma (2016.10.11), alxsah (2019.06.25), amogozov (2018.10.16), andreistanciu24 (2017.02.02), andresoro (2019.04.24), anuragbhd (2019.09.16), apaatsio (2015.09.14), apheleia (2016.05.29), apskim (2016.01.05), arjitc (2019.09.11), arshchimni (2019.09.16), ArthurHlt (2016.05.16), asaadmahmood (2015.06.19), avasconcelos114 (2017.11.09), awbraunstein (2019.02.13), ayadav (2017.01.18), AymaneKhouaji (2017.01.10), Banyango (2019.06.12), BastienDurel (2015.08.26), bbodenmiller (2017.09.27), bezumkin (2018.09.21), bjoernr-de (2016.12.20), BK1603 (2019.02.14), bnoggle (2019.07.12), bolariin (2019.05.27), bolecki (2016.11.15), bonespiked (2017.03.24), bradhowes (2017.03.23), brendanbowidas (2016.12.22), Brodan (2018.06.25), brunoqc (2016.04.08), carmo-evan (2019.05.29), ccbrown (2017.06.22), cdncat (2019.08.16), chahat-arora (2019.05.09), Charliekenney23 (2018.10.15), chclaus (2017.10.25), chengkun (2015.10.02), chengweiv5 (2015.06.27), cherealnice (2017.10.06), cherniavskii (2018.01.23), chetanyakan (2018.09.05), chikei (2016.09.15), chuck5 (2015.10.01), cifvts (2015.10.26), cimfalab (2018.09.17), cjbirk (2018.09.17), CometKim (2016.11.24), compaurum (2016.05.02), composednitin (2019.05.22), CoolTomatos (2018.03.06), CooperAtive (2019.04.15), cored (2018.10.17), coreyhulen (2015.06.15), courtneypattison (2019.02.19), cpanato (2016.11.14), cpfeiffer (2017.11.15), cpoile (2019.02.28), CrEaK (2017.01.19), crspeller (2015.06.15), csduarte (2016.11.05), CtrlZvi (2015.08.23), cyrilterets (2016.04.29), czertbytes (2018.10.15), d28park (2019.03.20), daizenberg (2016.02.25), dak425 (2019.06.10), danmaas (2018.09.24), DarrellRichards (2019.09.20), davidlu1997 (2016.05.02), dchukmasov (2019.03.06), deanwhillier (2018.10.22), debanshuk (2016.12.14), der-test (2017.05.08), deveshjadon98 (2017.11.10), digitaltoad (2016.10.31), dmitrysamuylovpharo (2018.08.29), doh5 (2017.03.29), dom3k (2019.01.30), doosp (2015.08.27, dos1701 (2018.10.08), drbaker (2015.10.01), driou (2015.09.22), dv29 (2019.01.31), ejachang (2019.02.21), ejm2172 (2016.01.04), enahum (2016.01.16), erikgui (2017.01.13), erikthered (2015.12.11), esdrasbeleza (2019.09.17), esethna (2015.09.17), eshyong (2019.07.15), evan-a-a (2019.05.02), evelikov (2018.06.28), faebser (2015.07.07), farhadab (2019.05.24), favadi (2017.02.02), feliciousx (2016.03.15), firstrow (2015.06.24), florianorben (2015.10.11), fraziern (2016.11.17), freerider7777	(2019.09.26), ftKnox (2017.06.22), FurmanovD (2018.09.17), gabel (2015.11.05), gabrieljackson (2019.01.29), GianOrtiz (2019.05.30), gig177 (2017.02.01), giorgosdi (2019.05.17), girishso (2015.10.18), GitHubJasper (2018.03.06), gnufede (2018.03.01), goku321 (2019.06.26), goofy-bz (2016.05.12), gramakri (2016.01.12), grundleborg (2016.09.30), gstraube (2017.04.17), GuillaumeAmat (2016.02.25), gupsho (2018.10.03), guydemi (2018.04.12), gvengel (2018.08.28), h2oloopan (2017.11.07), haikoschol (2015.06.24), Hanzei (2018.08.18), happygaijin (2019.04.05), harshavardhana (2015.10.26), harshilsharma63 (2018.09.26), hauschke (2016.04.21), hector2 (2019.09.11), herooftimeandspace (2018.07.24), hjf288 (2015.12.23), hmhealey (2015.07.03), Hobby-Student (2019.03.23), HugoGiraudel (2016.08.12), hvhallmann (2019.07.23), hvnsweeting (2015.12.25), Hyaxia (2019.07.07), Hyeongmin-Kwon (2017.11.02), Inconnu08 (2019.06.10), insin (2016.05.09), iomodo (2019.05.14), irbrad (2019.08.01), IshankGulati (2019.05.26), it33 (2015.06.15), ivanaairenee (2019.05.15), j8r (2019.03.27), jasimmons (2018.10.22), jasonblais (2015.08.27), JayaKrishnaNamburu (2018.10.15), jazzzz (2017.01.13), jdhoek (2015.11.10), jedisct1 (2015.06.24), JeffSchering (2017.03.02),  jerryfireman (2019.05.17), jesperhansen17 (2019.05.29), jespino (2017.09.14), JessBot (2015.09.08), jfrerich (2019.02.28), jkl5616 (2019.05.27), jkurian (2018.06.29), jlevesy (2018.10.15), joannekoong (2017.02.20), johnbellone (2019.04.30), johncoleman83 (2017.10.20), jonany (2019.06.26), jonathanwiesel (2015.11.20), joonsun-baek (2016.08.22), joshuabezaleel (2019.07.17), jostyee (2017.02.17), jurgenhaas (2017.01.20), justinwyer (2017.04.24), justyns (2015.09.29), jvasallo (2015.10.16), JVasky (2019.04.09), jwilander (2015.06.15), kaakaa (2016.09.16), keaton185 (2019.07.08), kelvintyb (2019.03.21), kemenaran (2018.02.02), KenmyZhang (2017.08.09), kennethjeremyau (2018.06.29), kernicPanel (2015.10.31), KerryAlsace (2018.11.02), kevinetienne (2019.06.26), kevynb (2016.06.10), kh0r (2015.12.09), khawerrind (2016.12.22), khoa-le (2016.02.02), kim95175 (2019.06.25), KishoreFartiyal (2017.11.09), kkamdooong (2017.06.29), kkirsche (2019.05.07), klingtnet (2018.10.03), knrt10 (2018.10.31), kongr45gpen (2018.09.11), kosgrz (2018.12.10), koukouloforos (2019.02.23), koxen (2018.02.20), krjn (2019.06.19), kulak-at (2017.05.09), LAndreas (2015.09.07), lassimus (2019.06.25), laur89 (2017.02.17), layzerar (2015.10.15), leblanc-simon (2018.03.01), letsila (2017.10.10), lfbrock (2015.07.30), lindalumitchell (2017.03.07), lindy65 (2015.08.27), Lisenish (2019.08.31), lmikaellukerad (2018.03.06), loafoe (2016.01.19), lologarithm (2018.10.04), longsleep (2017.10.06), lpadgett (2019.07.01), Lumexralph (2019.09.14), lurcio (2019.06.02), m3phistopheles (2019.03.14), manland (2019.02.06), manusajith (2015.09.03), MartB (2019.03.01), marianunez (2019.05.15), maruTA-bis5 (2016.05.07), matshch (2019.05.09), MatthewDorner (2019.08.29), matthewshirley (2019.08.09), mcmillhj (2015.10.21), metanerd (2019.08.19), MirlanMaksv (2019.02.06), MerlinDMC (2018.08.09), mfpiccolo (2016.11.04), mgdelacroix (2019.01.28), michaelkochub (2018.08.30), mickmister (2018.11.08), mikaoelitiana (2017.01.13), mikelinden1 (2017.12.01), mikroskeem (2018.09.27), mjthomp95 (2019.06.17), mkraft (2017.11.07), mishimi (2015.08.31), mogul (2017.11.24), mojicaj (2018.10.13), moonmeister (2017.06.29), morenoh149 (2017.03.01), mounicapaladugu (2019.06.06), mpoornima (2017.02.26), mstoli (2019.04.26), mukulrawat1986 (2018.10.13), MusikPolice (2017.07.24), nafisfaysal (2019.06.28), nfriend (2019.09.07), nickago (2015.06.28), niklabh (2019.09.10), nlowe (2019.01.24), noxer (2019.07.16), npcode (2015.11.23), odontomachus (2018.05.07), oliverJurgen (2019.03.14), olivierperes (2016.04.20), ollykel (2019.07.02), optimistiks (2015.10.12), pan-feng (2017.02.14), paranbaram (2016.08.22), patniharshit (2018.11.26), patrickkang	(2019.09.12), patterns (2019.06.28), pawelad (2016.01.18), pbitty (2019.09.19), peasead (2016.03.10), pepf (2016.11.17), phillipahereza (2019.07.03), philippe-granet (2018.03.12), phrix32 (2016.08.17), pieterlexis (2017.06.29), pietroglyph (2018.07.05), piperRyan (2019.06.06), pjgrizel (2016.03.24), Ppjet6 (2017.07.31), pradeepmurugesan (2018.07.04), prapti (2019.04.25), prixone (2017.03.30), pruthvip (2017.08.21), pstonier (2015.09.25), qichengzx (2018.03.19), QuantumKing (2017.11.14), RajatVaryani (2019.06.20), rajiv-k (2019.07.11), ralder (2015.06.26), raphael0202 (2016.11.14), reflog (2019.03.27), rexredinger (2019.07.02), rgarmsen2295 (2015.06.15), rickbatka (2017.10.17), rodrigocorsi2 (2016.02.09), rompic (2015.11.10), Rudloff (2016.09.13), ruzette (2017.02.05), rvillablanca (2019.05.30), R-Wang97 (2016.08.19), rwillmer (2016.11.01), ryantm (2017.10.10), ryoon (2016.03.24), s4kh (2018.10.21), samogot (2016.04.28), sandlis (2018.11.19), santos22 (2017.10.27), sapnasivakumar (2019.01.14), saturninoabril (2017.01.23), sbishel (2019.09.23), ScriptAutomate (2016.06.01), scottleedavis (2018.11.15), senk (2017.02.07), SergeyShpak (2018.10.23), sexybern (2015.10.05), SezalAgrawal (2019.06.27), Sheshagiri (2019.01.15), shieldsjared (2016.08.22), sij507 (2019.08.23), simon0191 (2017.04.17), S4KH (2016.10.21), sjmog (2015.09.23), sonasingh46 (2018.10.05), sowmiyamuthuraman (2019.09.10), srkgupta (2019.06.26), stasvovk (2015.10.12), streamer45 (2019.06.26), steevsachs (2019.06.27), stupied4ever (2016.03.22), sudheerDev (2017.08.21), tamtamchik (2015.11.26), tarikeshaq (2019.06.23), tauu (2019.02.05), tbalthazarte (2015.05.17), tejashreecd (2019.09.19), tengis617 (2019.04.23), tkbky (2017.10.26), tehraven (2016.06.01), tgkouras (2019.09.11), theblueskies (2018.07.18), thekiiingbob (2019.03.07), therealpuneeth20 (2019.04.25), ThiefMaster (2018.06.19), thiyagaraj (2016.06.15), thomas9987 (2015.10.01), thomchop (2016.10.26), threepwood-mm (2019.08.29), timlyo (2016.02.19), tjuerge (2017.05.31), tmuwandi (2015.10.01), tomasmik (2019.06.08), tomitm (2015.10.06), tomocy (2019.01.12), toyorg (2015.10.28), trozz (2015.09.12), Tsynapse (2015.11.06), tylarb (2019.04.22), tyvsmith (2018.10.25), usmanarif (2016.05.03), uturkdogan (2018.05.15), vaithak (2018.11.05), valentijnnieman (2019.09.16), VeraLyu (2017.03.15), vinnymac (2015.11.01), viot (2015.10.02), vladikoff (2015.11.20), VolatianaYuliana (2019.05.07), vordimous (2018.02.07), Vorlif (2018.03.13), VPashkov (2018.10.24), waseem18 (2018.10.03), willdot (2019.06.06), willstacey (2015.08.27), Willyfrog (2019.06.15), wget (2016.10.17), Whiteaj36 (2017.08.03), Wipeout55 (2019.05.20), XinyueWang94 (2018.03.06), yeoji (2017.10.16), yth0625 (2017.11.07), yumenohosi (2016.05.04), yuvipanda (2015.11.01), Zackify (2015.08.29), Zaicon (2017.02.20), Zapix (2017.10.05), zeroimpl (2018.11.08), zetaab (2018.09.10), ZJvandeWeg (2017.01.09), zkry (2019.06.12)
```

Comment (justinegeffen) [reply site/content/contribute/team_contributions/_index.md:28]: Fixed some spacing and added an opening "(". Hopefully didn't create any erroneous additional spaces.

**Comment by justinegeffen** [comment]
This is awesome, @jasonblais! :)

File: site/content/contribute/team_contributions/_index.md [hunk 2]
```diff
@@ -9,22 +9,44 @@ Mattermost is an open source Slack-alternative that helps teams stay in sync, wi
 
 The project is developed by a global community of contributors and is maintained and supported by a distributed company, [Mattermost Inc.](https://about.mattermost.com/company/), based in Palo Alto, California.
 
+If you notice an error or omission in the list, please email community[at]mattermost.com so we can correct the issue.
+
+## Community Advocates
+
+Community Advocates by full name (first contribution date) are alphabetically listed below. The list is updated regularly (last updated June 26, 2020), and includes contributions to the mattermost.com/blog via an article.
+
+Join other community advocates by joining [our Mattermost Superstars program](https://mattermost.com/community/superstars).
```
Comment (hanzei) [inline site/content/contribute/team_contributions/_index.md:18]: The link returns 404. Should it be setup before merging?

File: site/content/contribute/team_contributions/_index.md [hunk 3]
```diff
@@ -9,22 +9,44 @@ Mattermost is an open source Slack-alternative that helps teams stay in sync, wi
...
+
+## Solution Contributors
```
Comment (hanzei) [inline site/content/contribute/team_contributions/_index.md:30]: What are `Solution Contributors`?

File: site/content/contribute/team_contributions/_index.md [hunk 4]
```diff
@@ -9,22 +9,44 @@ Mattermost is an open source Slack-alternative that helps teams stay in sync, wi
...
+
+## Documentation Contributors
+
+To be added.
+
+## Security Research Contributors
```
Comment (hanzei) [inline site/content/contribute/team_contributions/_index.md:46]: :100:

File: site/content/contribute/team_contributions/_index.md (hunk 3 above)
Comment (jasonblais) [reply site/content/contribute/team_contributions/_index.md:30]: Those who have created an integration, installer, package manager, plugin and more. Definitely open to other naming as well

File: site/content/contribute/team_contributions/_index.md (hunk 2 above)
Comment (jasonblais) [reply site/content/contribute/team_contributions/_index.md:18]: Good call, I'll remove the link for now until the page is published

File: site/content/contribute/team_contributions/_index.md (hunk 3 above)
Comment (hanzei) [reply site/content/contribute/team_contributions/_index.md:28]: Maybe `Integration Contributors` as all these items are listed on https://integrations.mattermost.com/

**Pull request merged**